Project.toml

```
@@ -1,12 +1,12 @@
 name = "Tau"
 uuid = "c544e3c2-d3e5-5802-ac44-44683f340e4a"
-version = "1.0.0"
+version = "1.1.0"
```

To be on the safe side since types change and hence possibly dispatches are broken:
```suggestion
version = "2.0.0"
```